lib/digital_public_works_web/plugs/auth.ex

```diff
@@ -9,13 +9,17 @@ defmodule DigitalPublicWorks.Plugs.Auth do
     if conn.assigns[:current_user] do
       conn
     else
-      current_user = if user_id = get_session(conn, :current_user_id) do
-        Accounts.get_user!(user_id)
-      end
+      current_user =
+        if user_id = get_session(conn, :current_user_id) do
+          try do
+            Accounts.get_user!(user_id)
```

Comment:
Add an `Accounts.get_user(user_id)` to the accounts context that returns {:ok, user} or {:error, err} and use a case.